rmw_connext_cpp/src/rmw_take.cpp

```diff
@@ -248,6 +248,13 @@ _take_sequence(
     return RMW_RET_ERROR;
   }
 
+  if (count > (std::numeric_limits<DDS_Long>::max)()) {
```

Comment:
Looks like this introduces a new compiler warning on linux https://ci.ros2.org/job/ci_linux/10551/gcc/new/source.bde60058-8945-4cc4-ae0d-abd08de67f16/#252

`comparison of integer expressions of different signedness: ‘size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare] 252 | if (count > (std::numeric_limits::max)()) { | `


Though it's odd that the warning says `(std::numeric_limits::max)()` without the template type

Reply:
https://github.com/ros2/rmw_connext/pull/420